src/libraries/Native/Unix/System.Security.Cryptography.Native/CMakeLists.txt

```diff
@@ -75,11 +75,6 @@ add_library(System.Security.Cryptography.Native.OpenSsl-Static
     $<TARGET_OBJECTS:objlib>
 )
 
-# Disable the "lib" prefix.
-set_target_properties(System.Security.Cryptography.Native.OpenSsl PROPERTIES PREFIX "")
-
-# Disable the "lib" prefix and override default name
-set_target_properties(System.Security.Cryptography.Native.OpenSsl-Static PROPERTIES PREFIX "")
```

Comment:
I'm not too familiar with this so probably @bartonjs should take a look, but looks like in here you are also changing the override of the default name

Reply:
> and override default name

this part of the comment refers to the line below. There are two variants of this library produced by this script: shared and static. We have prefix override for each which this delta is removing.

Reply:
Yes this change is correct. We could however preserve the ` override default name` part of the comment for the next line which sets the OUTPUT_NAME on the static lib.